cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,102 @@
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
+  let employee;
 
+  beforeEach(() => {
+    cy.task('generateEmployee').then((generateEmployee) => {
+      employee = generateEmployee;
+    });
+    cy.visit('/');
+  });
+
+  it('should contain pagination', () => {
+    cy.get('.pagination-bottom').should('exist');
+    cy.get('.pagination-bottom').should('contain', 'Previous');
+    cy.get('.pagination-bottom').should('contain', 'Page');
+    cy.get('.pagination-bottom').should('contain', 'of');
+    cy.get('.pagination-bottom').should('contain', 'rows');
+    cy.get('.pagination-bottom').should('contain', 'Next');
+  });
+
+  it('should provide the ability to select count of rows', () => {
+    cy.createUser(employee);
+    cy.createUser(employee);
+    cy.createUser(employee);
+    cy.get('.select-wrap').should('contain', '10 rows');
+    cy.get('.select-wrap').should('contain', '20 rows');
+    cy.get('.select-wrap').should('contain', '5 rows');
+    cy.get('select').select('5');
+    cy.get('.-next').click();
+    cy.get('[aria-label="jump to page"]').should('have.value', '2');
+    cy.get('.-previous').click();
+    cy.get('[aria-label="jump to page"]').should('have.value', '1');
+  });
+
+  it('should provide the ability to add new worker', () => {
+    cy.createUser(employee);
+    cy.get('.ReactTable').should('contain', employee.firstName)
+      .and('contain', employee.lastName)
+      .and('contain', employee.email)
+      .and('contain', employee.age)
+      .and('contain', employee.salary)
+      .and('contain', employee.department);
+  });
+
+  it('should provide the ability to delete a worker', () => {
+    cy.createUser(employee);
+    cy.get('#delete-record-4').click();
+    cy.get('.ReactTable').should('not.contain', employee.firstName)
+      .and('not.contain', employee.lastName)
+      .and('not.contain', employee.email)
+      .and('not.contain', employee.age)
+      .and('not.contain', employee.salary)
+      .and('not.contain', employee.department);
+  });
+
+  it('should provide the ability to delete all workers', () => {
+    cy.get('#delete-record-1').click();
+    cy.get('#delete-record-2').click();
+    cy.get('#delete-record-3').click();
+    cy.contains('No rows found').should('exist');
+  });
+
+  it('should provide the ability to find a worker' +
+    'edit it and validate data', () => {
+    cy.get('#searchBox').type('Cierra');
```

Comment:
The logic is correct. But what if such a user is not found? It will be more reliable to add your user and then find and edit it